teps/0091-trusted-resources.md

```diff
@@ -377,13 +386,44 @@ metadata:
   name: feature-flag
   namespace: tekton-pipelines
 data:
-  verification-policy: enforce
+  enable-trusted-resource: true
 ```
 
-`verification-policy`. (Optional, `enforce`, `warn` or `skip`, default `skip`):
- * `enforce`: Failing verification will mark the taskruns/pipelineruns as failed.
- * `warn`: Log warning but don't fail the taskruns/pipelineruns.
- * `skip`: Directly skip the verification.
+`enable-trusted-resource`. (Optional, `true` or `false`, default `false`):
+ * `true`: Failing verification will mark the taskruns/pipelineruns as failed.
+ * `false`: Directly skip the verification.
+
+
+ ```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-trusted-resources
+  namespace: tekton-pipelines
+data:
+  no-match-policy: "deny"
+```
+
+`no-match-policy`. (Optional, `allow`, `deny` or `warn`, default `deny`):
```

Comment:
Maybe a flow chart is useful here.
I presume if enable-trusted-resource is false then entire trusted resource code is not touched, so no messaging to the user.
If true and no-match-policy is allow then resources without policies will be allow, but will there be a message that it was allowed? Will a condition be set here? How does user know that the resource did not have a verification policy but was still allowed? Maybe another Reason in conidition?
If true and no-match-policy is warn then resources without policies will be warned. "Warn" is defined as set the condition Reason to "podconvert.ReasonResourceVerificationFailed," ?

Reply:
Thanks!